### PR TITLE
create pr against sunshine

### PR DESCRIPTION
**Pull request by ReenigneArcher**
## Description

- create a pr against sunshine
  - This will update all ffmpeg submodules in a single PR, whereas dependabot creates a PR for every submodule branch, which generates a ton of workflow runs for Sunshine. This PR aims to reduce the workflow runs. Additionally it will be much faster to merge the changes into Sunshine as checks will only need to pass on a single PR instead of multiple.

📝 This depends on a fork of an action to enable PRs against another repository. Once this PR is merged, the workflow can be changed back to the original maintainer. https://github.com/repo-sync/pull-request/pull/69

Todo:
- [x] enable only on specific events (i.e. merge to `master`)
- [x] enable job needs
- [x] test that PR is created properly (should run immediately after creating this PR)
- [x] specified PR labels are not added
- [x] do not automerge these PRs -> https://github.com/LizardByte/.github/issues/152

## Type of Change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency update (updates to dependencies)
- [ ] Documentation update (changes to documentation)
- [x] Repository update (changes to repository files, e.g. `.github/...`)

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated the in code docstring/documentation-blocks for new or existing methods/components

## Branch Updates
LizardByte requires that branches be up-to-date before merging. This means that after any PR is merged, this branch
must be updated before it can be merged. You must also
[Allow edits from maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I want maintainers to keep my branch updated

### CODE REVIEW

**Review by brad-richardson — DISMISSED** [review]

File: .github/workflows/build-ffmpeg.yml [hunk 1]
```diff
@@ -551,3 +553,79 @@ jobs:
...
+            ffmpeg-linux-aarch64
+            ffmpeg-linux-x86_64
+            ffmpeg-macos-aarch64
+            ffmpeg-macos-x86_64
+            ffmpeg-windows-x86_64
+          VAREOF
+          )
+
+          echo "::group::git checkout"
+          git checkout -b ${{ env.update_branch }} || git checkout ${{ env.update_branch }}
```
Comment (ReenigneArcher) [inline .github/workflows/build-ffmpeg.yml:593]: how to properly rebase nightly onto this branch without adding additional commits?

Comment (psyke83) [reply .github/workflows/build-ffmpeg.yml:593]: `git rebase origin/nightly` ?

Comment (psyke83) [reply .github/workflows/build-ffmpeg.yml:593]: If you can check locally, `git rebase -i origin/nightly` will interactively show the commits (which you need to confirm before proceeding).

**Comment by ReenigneArcher** [comment]
An issue was discovered.

The destination branch on sunshine still existed (although the previous PR was merged), and a new PR was created.

The destination branch looked like this when the PR was created.
image

Missing new commits from `nightly`.
image

**Pull request merged**